src/main/java/org/myrobotlab/codec/CodecUtils.java

```diff
@@ -34,51 +24,210 @@
 import org.yaml.snakeyaml.Yaml;
 import org.yaml.snakeyaml.constructor.Constructor;
 
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
-import com.google.gson.internal.LinkedTreeMap;
+import java.io.*;

```

Comment:
Should CTRL+SHIFT+O to organize imports, wildcards are not needed

Reply:
Whoops, Intellij did that on an autoformat, I'll have to adjust my settings

Reply:
Should be fixed now